src/hardware/input/mouse_common.cpp

```diff
@@ -90,16 +91,58 @@ uint8_t MOUSE_GetDelayFromRateHz(const uint16_t rate_hz)
 
 float MOUSE_ClampRelativeMovement(const float rel)
 {
+	constexpr float Max = 2048.0f;
 	// Enforce sane upper limit of relative mouse movement
-	return std::clamp(rel, -2048.0f, 2048.0f);
+	return std::clamp(rel, -Max, Max);
+}
+
+float MOUSE_ClampWheelMovement(const float rel)
+{
+	constexpr float Max = 127.0f; // so that it always fits into int8_t
```

Comment:
Minor: no long end of the line comments, please.

Reply:
OK, changed.